cmd/config-downloader/downloader.go

```diff
@@ -125,13 +125,15 @@ func main() {
 	if inputConfig != "" {
 		f, err := os.Open(inputConfig)
 		if err != nil {
-			fmt.Printf("E! Failed to open Common Config: %v\n", err)
-			panic(err)
+			errorMessage := fmt.Sprintf("E! Failed to open Common Config: %v\n", err)
+			log.Printf(errorMessage)
```

Comment:
So, two things.
1. I was reading up on the differences between `fmt.Print()` and `log.Print()` and from what I saw online, by default, the `log` package writes to stderr whereas `fmt` writes to stdout. **This would result in a change in behavior**. 
2. If we determine it's okay to accept the above change in behavior, this could be simplified to 
```suggestion
			log.Panicf("E! Failed to open Common Config: %v\n", err)
```